File: problem-sets/pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //get user input
    int n;
    do
    {
        n = get_int("Enter size of pyramid: ");
    }
    while (n < 1 || n > 8);

    //loop
    for (int i = 0; i < n; i++)
    {
        //print spaces
        for (int s = (n - i - 2); s >= 0; s--)
        {
            printf(" ");
        }
        //print ladders
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: problem-sets/pset6/python/mario/mario.py
from cs50 import get_int

#pyramid function
def pyramid():
    num = usernum()
    i = 0
    #line loop
    for i in range(0, (num)):
        s = num - i - 1
        #space loop
        for sp in range(s, 0, -1):
            print(" ", end="")
        j = 0
        #pound sign loop
        for j in range(0, i + 1):
            print("#", end="")
        print("")

# ask user size of pyramid
def usernum():
    while True:
        num = get_int("Enter size of pyramid: ")
        if num > 0 and num < 9:
            break
    return num


pyramid()